src/main/java/net/smartcosmos/dao/relationships/util/UuidUtil.java

```diff
@@ -73,4 +73,8 @@ public static UUID getNewUuid() {
 
         return UUID.fromString(sortedUuidString);
     }
+
+    public static String getNewUuidAsString() {
```

Comment:
I'm frankly not convinced of the value of this method.  It is trivial thing to call `toString()`.